hikari/impl/entity_factory.py

```diff
@@ -1406,19 +1406,19 @@ def deserialize_invite_with_metadata(self, payload: data_binding.JSONObject) ->
     ##################
 
     def deserialize_partial_message(self, payload: data_binding.JSONObject) -> message_models.PartialMessage:
-        author: undefined.UndefinedOr[user_models.User] = undefined.UNDEFINED
-        if "author" in payload:
-            author = self.deserialize_user(payload["author"])
+        assert (
+            "author" in payload
```

Comment:
This isn't included in message updates triggered by embed workers. 
See the following example
```json
{
    "id": "759533107606585354",
    "embeds": [
        {
            "video": {
                "width": 1280,
                "url": "https://www.youtube.com/embed/Eak9JO1zO2U",
                "height": 720
            },
            "url": "https://www.youtube.com/watch?v=Eak9JO1zO2U",
            "type": "video",
            "title": "Conduite \u00e0 tenir la PLS : Position Lat\u00e9rale de S\u00e9curit\u00e9 - Protectio...",
            "thumbnail": {
                "width": 1280,
                "url": "https://i.ytimg.com/vi/Eak9JO1zO2U/maxresdefault.jpg",
                "proxy_url": "https://images-ext-1.discordapp.net/external/IMPARnTHG9KAVzy1MrY3P6ZxrGkAXuIDNTVN3lFHfLI/https/i.ytimg.com/vi/Eak9JO1zO2U/maxresdefault.jpg",
                "height": 720
            },
            "provider": {
                "url": "https://www.youtube.com",
                "name": "YouTube"
            },
            "description": "Position Lat\u00e9rale de S\u00e9curit\u00e9 : PLS. \nD\u00e9monstration comment\u00e9e en temps r\u00e9el par la Protection Civile de Paris\n\nD\u00e9couvrez sur notre site Internet toutes nos formations www.protectioncivile.org\n\n\nPlus de 480 b\u00e9n\u00e9voles au sein de la Protection Civile de Paris ; pourquoi pas vous ...",
            "color": 16711680,
            "author": {
                "url": "https://www.youtube.com/channel/UCtM2N8EnpnlX4vnONCkSmWQ",
                "name": "Protection Civile Paris Seine"
            }
        }
    ],
    "channel_id": "574921092926537729",
    "guild_id": "574921006817476608"
}
```

Reply:
really

ffs.

Reply:
Fixed.